tests/transforms/test_split_non_commuting.py

```diff
@@ -61,7 +62,10 @@ def test_commuting_group_no_split(self, mocker, meas_type):
             meas_type(op=qml.PauliZ(0))
             meas_type(op=qml.PauliX(1))
             meas_type(op=qml.PauliZ(2))
-            meas_type(op=qml.PauliZ(0) @ qml.PauliZ(3))
+            if convert_to_opmath:
+                meas_type(op=qml.prod(qml.Z(0), qml.Z(3)))
+            else:
+                meas_type(op=qml.PauliZ(0) @ qml.PauliZ(3))
```

Comment:
```suggestion
                meas_type(op=qml.operation.Tensor(qml.PauliZ(0), qml.PauliZ(3)))
```
Since we want to make the switch soon